tests/test_pipeline.py

```diff
@@ -25,12 +25,12 @@
 
 @pytest.mark.parametrize('potato_variation', ['potato', ' potato ', ' potato', 'potato ', ' POTATO ', ' PoTatO '])
 def test_parse_str(potato_variation: str) -> None:
-    ta_lower = TypeAdapter(Annotated[str, validate_as(...).str_strip().str_lower()])
+    ta_lower = TypeAdapter[str](Annotated[str, validate_as(...).str_strip().str_lower()])
```

Comment:
What error do we get if we don't explicitly parametrize the `TypeAdapter`? Is there any way for said type to be inferred from the annotation?

Reply:
Probably not haha I'm guessing you would have already handled that, but just wanted to check. Guessing it's the whole generic can of worms.

Reply:
Not until the TypeForm PEP gets merged...

I thought it used to end up as `TypeAdapter[Any]` but it seems like it doesn't now. Maybe because of https://github.com/pydantic/pydantic/commit/8a896c51561fe9aa51cd7a8adabdb79b70bd5bce?

Reply:
Looks like https://github.com/pydantic/pydantic/commit/953bbea8636742232dc659f8d92936c4b1ea94c1#diff-2695629fb80cb6eb6be5260a4611b3b831a7ebd53041d01f37c0d67dd55928fa is the one that broke it

Reply:
https://github.com/pydantic/pydantic/pull/9680

Reply:
Hum, in my https://github.com/pydantic/pydantic/pull/9570 fix I kept the `__init__` so that users would be forced to parametrize the `TypeAdapter` if the type checker can't use the first overload. But having an implicit fallback to `TypeAdapter[Any]` seems good as well